.teamcity/delivery/production/manifest/PublishHubLatest.kts

```diff
@@ -0,0 +1,70 @@
+package delivery.production.manifest
+
+import utils.ImageInfoRepository
+import utils.dsl.general.teamCityImageBuildFeatures
+import utils.dsl.general.teamCityProdImagesSnapshot
+import utils.dsl.steps.publishManifest
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildTypeSettings
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+
+/**
+ * TODO: Merge with 'publish_hub_version'?
+ */
+object publish_hub_latest : BuildType({
```

Comment:
Please, note that I've intentionally left `TODO` comment - in my opinion, the configurations could be safely merged, however, I don't rule out the probability they've been separated on purpose.

Reply:
We often publish the version a little earlier (nobody will try to pull them the a not-yet-released-version tag) and the "latest" only when all other activities around the release are done. So these should be two different build configurations.